.github/workflows/vercel.yml

```diff
@@ -21,6 +22,12 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Set up node
```

Comment:
I see that these changes are exist in develop, but for some reason they are highlighted here as addition